painters/model_def_standard.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+import keras
+import numpy as np
+import tensorflow as tf
+from keras import backend as K
+from keras.datasets.cifar import load_batch
+from keras.layers import (Activation, Conv2D, Dense, Dropout, Flatten,
+                          MaxPooling2D)
+from keras.losses import categorical_crossentropy
+from keras.metrics import categorical_accuracy
+from keras.models import Sequential
+from keras.optimizers import Adam
+from keras.utils.data_utils import get_file
+
+import pedl
+from pedl.data import ArrayBatchLoader
+from pedl.frameworks.keras import KerasTrial
+from pedl.frameworks.keras.data import KerasDataAdapter
+from pedl.frameworks.util import elementwise_mean
+from sklearn.metrics import roc_auc_score
+
+from train_cnn import _cnn, IMGS_DIM_3D, BATCH_SIZE, SOFTMAX_SIZE
+from data_provider import load_organized_data_info, train_val_dirs_generators
+from validation import _create_pairs_generator, IMGS_DIM_1D
+from utils import pairs_dot
+
+import objgraph
+
+def categorical_error(y_true, y_pred):
+    return 1. - categorical_accuracy(y_true, y_pred)
+
+
+class PainterTrial(KerasTrial):
+    def __init__(self, hparams):
+        super().__init__(hparams)
+
+        self.kernel_size = pedl.get_hyperparameter("kernel_size")
+        self.dropout = pedl.get_hyperparameter("dropout")
+        self.pool_size = pedl.get_hyperparameter("pool_size")
+        self.L2_REG = pedl.get_hyperparameter("L2_REG")
```

Comment:
Capitalized names are usually reserved for globals in Python -- class attributes should be lowercased.